testing/helpers/wrappers/tabsWrappers.js

```diff
@@ -32,13 +32,13 @@ export class TestTabsWrapper {
     }
 
     checkTabsWithoutScrollable() {
-        const $tabs = this._$tabs.find('> .dx-tabs-wrapper > .dx-tab');
+        const $tabs = this._$tabs.find('.dx-tabs-wrapper > .dx-tab');
         QUnit.assert.equal($tabs.length, this._tabs.option('items').length, 'tabs are rendered');
         this._checkTabsContent($tabs);
     }
 
     checkTabsWithScrollable() {
-        const $tabs = this._$tabs.find('> .dx-tabs-scrollable .dx-scrollable-content > .dx-tabs-wrapper > .dx-tab');
+        const $tabs = this._$tabs.find('> .dx-tabs-scrollable .dx-scrollable-content .dx-tabs-wrapper > .dx-tab');
```

Comment:
Let's extract such changes to a new PR and review why we need them.

Reply:
This changes needs us while our 'stopper' problems not fixed. After that problems will fix I am going to revert such changes.